File: antlr_c/antlr_c/Clases/Directiva.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace antlr_c.Clases
{
    public class Directiva
    {
        private int vida;
        private int tiempo;
        private char unidad;

        public Directiva(int vida, int tiempo, char unidad)
        {
            this.vida = vida;
            this.tiempo = tiempo;
            this.unidad = unidad;
        }

        public int getVida()
        {
            return this.vida;
        }

        public int getTiempo()
        {
            return this.tiempo;
        }
        public char getUnidad()
        {
            return this.unidad;
        }
    }
}

File: pl_cup/Directiva.java
public class Directiva {
    private Integer vida;
    private Integer tiempo;
    private char unidad;

    public Directiva(Integer vida, Integer tiempo, char unidad){
        this.vida=vida;
        this.tiempo=tiempo;
        this.unidad=unidad;
    }

    public Integer getVida(){
        return this.vida;
    }

    public Integer getTiempo(){
        return this.tiempo;
    }
    public char getUnidad(){
        return this.unidad;
    }
    
}
